lua/bufferline.lua

```diff
@@ -27,12 +25,15 @@ local bufferline = {}
 --- @param user_config? table
 --- @return nil
 function bufferline.setup(user_config)
-  -- Show the tabline
-  set_option('showtabline', 2)
+  if type(user_config) ~= "table" then
```

Comment:
More inclusive condition, such that passing `true`, `4`, `'foo'`, `nil`, etc to `setup` won't cause an error.